public/manifest.json

```diff
@@ -1,6 +1,6 @@
 {
-  "short_name": "React App",
-  "name": "Create React App Sample",
+  "short_name": "Vocascan | Web",
```

Comment:
why do you use `| Web` I can imagine, that this doesnt look good, if you add the page to your homescreen on your mobile phone. I think vocascan should be enough. But you can try if you only set short name to that, if it will be used as an app name on the mobile phones homescreen.